unitTests/ui/DynamicFacet/DynamicFacetTest.ts

```diff
@@ -275,14 +275,32 @@ export function DynamicFacetTest() {
     });
 
     it(`when calling reset
-      when there are no active values
-      should not clear and rerender values`, () => {
+    when there is any active value
+    should set the flag for preventAutoSelect`, () => {
+      mockFacetValues[0].state = FacetValueState.selected;
+      initializeComponent();
+
+      test.cmp.reset();
+
+      expect(test.cmp.dynamicFacetQueryController.buildFacetRequest().preventAutoSelect).toBe(true);
+    });
+
+    it(`when calling reset
+    when there are no active values
```

Comment:
`when there are no active values` guess this means it is not being mocked as far as the test is concerned?

Reply:
Yeah there's by default every values are inactive, we don't need to do anything special in the test setup